pkgs/development/python-modules/greenlet/default.nix

```diff
@@ -12,7 +12,6 @@ buildPythonPackage rec {
   pname = "greenlet";
   version = "2.0.2";
   format = "setuptools";
-  disabled = isPyPy; # builtin for pypy
```

Comment:
I think this would have been worth keeping.

Reply:
No, because it also disables reverse dependencies, while setting it to null allows reverse depencies to build.

Reply:
Well, we could've kept it, but there is no point, since it won't be called from `pypy3Packages` anymore.